test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -348,6 +348,38 @@ abstract class LorisIntegrationTest extends TestCase
         $this->DB->run('SET foreign_key_checks =1');
     }
 
+    /**
```

Comment:
This will be removed once #6912 is merged. These are helper functions to test project permissions.

Reply:
This was removed